client/src/component/page/ErrorPage.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { JSX } from 'react';
+import { useLocation } from 'react-router-dom';
+import { ERROR_MESSAGES } from '../error/ErrorMessages';
+import { Location } from '@remix-run/router';
+
+const ErrorPage = (): JSX.Element => {
+  const location: Location = useLocation();
+
+  return (
+    <div
+      data-testid="error-page"
+    >
+      {location?.state?.errorMessage || ERROR_MESSAGES.INTERNAL_SERVER_ERROR}
```

Comment:
If an error message has been passed then that will be rendered otherwise it will just show the default internal server error page. For example if you navigate to a route other than `/`, `/form` than you will be re-directed to the error page but instead see the page not found error message.